src/CMakeLists.txt

```diff
@@ -181,20 +181,21 @@ install(TARGETS umf
 
 add_subdirectory(pool)
 
-# TODO: enable proxy_lib on Windows
-if(LINUX)
-    if(UMF_PROXY_LIB_BASED_ON_POOL STREQUAL SCALABLE)
-        set(PROXY_LIB_USES_SCALABLE_POOL ON)
-        set(PROXY_LIBS umf scalable_pool)
+if(LINUX OR WINDOWS)
```

Comment:
Now that proxy lib is available on linux and windows is this `if` necessary? Also, should we create a cmake option for building proxy lib?

Reply:
Yes, we have MACOSX too ... :-(
The option is not necessary as for now.